e2e/tests/stepDefinitions/manage/lostBed.ts

```diff
@@ -0,0 +1,56 @@
+import { Given, Then } from '@badeball/cypress-cucumber-preprocessor'
+import Page from '../../../../cypress_shared/pages/page'
+import BedspaceShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/bedspaceShow'
+import LostBedNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/lostBedNew'
+import newLostBedFactory from '../../../../server/testutils/factories/newLostBed'
+import lostBedFactory from '../../../../server/testutils/factories/lostBed'
+import LostBedShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/lostBedShow'
+
+Given("I'm marking a bedspace as void", () => {
```

Comment:
Appreciate I've not been fully consistent, but I think let's standardise on `Given("I mark a bedspace as void")` etc. for how we word these

Reply:
ok - as you say there are inconsistencies with other scenario descriptions but we can deal with those later.